app/Http/Transformers/ActionlogsTransformer.php

```diff
@@ -98,6 +100,13 @@ public function transformActionlog (Actionlog $actionlog, $settings = null)
                                         \Log::debug('custom fields do not match');
                                         $clean_meta[$fieldname]['old'] = "************";
                                         $clean_meta[$fieldname]['new'] = "************";
+
+                                        // Display the changes if the user is an admin or superadmin
+                                        if (Gate::allows('admin')) {
+                                            $clean_meta[$fieldname]['old'] = ($enc_old) ? unserialize($enc_old): '(blank)';
```

Comment:
Should we be using a translation string here?

Reply:
Probably

Reply:
I changed it to `''` instead to make it more consistent with the non-custom fields